modules/data-entry/src/main/frontend/src/dataHomepage/PermissionsButton.jsx

```diff
@@ -34,7 +34,7 @@ function PermissionsButton(props) {
     <React.Fragment>
       <Tooltip title="Set Permissions">
         <IconButton component="span" className={classes.titleButton} size="large">
-          <Lock fontSize={size ? size : "default"}/>
+          <Lock fontSize={size ?? "default"}/>
```

Comment:
The exact translation of the original statement is 
```suggestion
          <Lock fontSize={size || "default"}/>
```
I don't know if we'd ever get empty string as a parameter here, but if we do it would be a regression.